soc/ti/mspm0/mspm0g/Kconfig.soc

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2024 Texas Instruments
+# Copyright (c) 2025 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_MSPM0G
+	bool
+	select SOC_FAMILY_TI_MSPM0
+	help
+	  Enable support for TI MSPM0G series SoCs
+
+config SOC_MSPM0G1105
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G1106
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G1107
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G1505
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G1506
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G1507
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G1519
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G3105
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G3106
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G3107
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G3505
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G3506
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G3507
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_MSPM0G3519
+	bool
+	select SOC_SERIES_MSPM0G
+
+config SOC_SERIES
+	default "mspm0g" if SOC_SERIES_MSPM0G
+
+config SOC
+	default "MSPM0G1105" if SOC_MSPM0G1105
+	default "MSPM0G1106" if SOC_MSPM0G1106
+	default "MSPM0G1107" if SOC_MSPM0G1107
+	default "MSPM0G1505" if SOC_MSPM0G1505
+	default "MSPM0G1506" if SOC_MSPM0G1506
+	default "MSPM0G1507" if SOC_MSPM0G1507
+	default "MSPM0G1519" if SOC_MSPM0G1519
+	default "MSPM0G3105" if SOC_MSPM0G3105
+	default "MSPM0G3106" if SOC_MSPM0G3106
+	default "MSPM0G3107" if SOC_MSPM0G3107
+	default "MSPM0G3505" if SOC_MSPM0G3505
+	default "MSPM0G3506" if SOC_MSPM0G3506
+	default "MSPM0G3507" if SOC_MSPM0G3507
+	default "MSPM0G3519" if SOC_MSPM0G3519
```

Comment:
lowercase for the actual string e.g. `	default "mspm0g1105" if SOC_MSPM0G1105`

Reply:
Is It hard rule to use lowercase ?

Reply:
yes

Reply:
Zephyr SoC Porting Guide says "Notice that SOC_NAME is a pure upper case version of the SoC name." but string literals doesn't.

I will fix it.